packages/dredd/test/fixtures/json-schema-draft-7.apib

```diff
@@ -0,0 +1,32 @@
+FORMAT: 1A
+
+# Machines API
+
+# Group Machines
+
+# Machines collection [/machines]
+
+## Get Machines [GET]
+
+- Response 200 (application/json; charset=utf-8)
+
+    + Schema
+        
+            {
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "enum": ["bulldozer"]
```

Comment:
We could use `const` instead of `enum` because const was added in draft 6 or 7 so it can ensure that this is treated as a newer draft since it won't be valid in draft 3/4.

```suggestion
                            "const": "bulldozer"
```

Reply:
That's a wonderful idea. Will change.

Reply:
Changed in https://github.com/apiaryio/dredd/pull/1622/commits/dfd485938c2b764ff015b1acf28885e1af0a8f55.